packages/core/src/types/IViewport.ts

```diff
@@ -110,17 +110,76 @@ export type ReferencedImageRange = ViewReference & {
   referencedImageId: string;
 };
 
+/**
+ *  A referenced plane specifies a planar coordinate system
+ * The most degenerate case has a single point, and includes all planes
+ * which pass through that point.  For this case, if a view plane normal
+ * is required to display the point, the current view plane normal of the
+ * viewport is recommended, or the acquisition plane if there is no view plane
+ * normal.
```

Comment:
what is the case that we don't have view plane normal? there is always a vpn no?